libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -216,6 +218,14 @@ - (void)viewDidLoad
     NSString *hybridViewUserAgentString = [self sfHybridViewUserAgentString];
     [SFSDKWebUtils configureUserAgent:hybridViewUserAgentString];
     self.baseUserAgent = hybridViewUserAgentString;
+    
+    // Run a basic validation of the app config inputs.
+    SFSDKAppConfigValidationResult *validationResult = [_hybridViewConfig validate];
```

Comment:
The one oddity in this approach is that, when authentication is set to `true`, auth happens before any of the hybrid view takes off.  The net result is that you can authenticate, then end up in a validation failure because e.g. you specified an absolute URL for your start page.

I'm not sure how to get around that without a more significant refactor.  One approach could be to manage the validation step in `SalesforceSDKManager` itself.